Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -283,7 +283,8 @@ public class SchemeGenerator {
             launchAction: launchAction,
             profileAction: profileAction,
             analyzeAction: analyzeAction,
-            archiveAction: archiveAction
+            archiveAction: archiveAction,
+            wasCreatedForAppExtension: nil
```

Comment:
Sorry this was the bit that need the nil check. At the moment all this code doesn't do anything yet
```suggestion
            wasCreatedForAppExtension: scheme.wasCreatedForAppExtension ? scheme.wasCreatedForAppExtension : nil,
```
To test this, could you set a property on one of the schemes in TestProject fixture, and commit the diffs

Reply:
Can I add new target, e.g: NotificationService to Tests/Fixtures/TestProject/project.yml or I should use one of available extension target. For those target seems don't require wasCreatedForAppExtension to be YES.

Reply:
If there are no relevant app extensions, feel free to add one 👍